xapian-applications/omega/docs/omegascript.rst

```diff
@@ -222,10 +222,10 @@ $fmt
 $freq{term}
 	frequency of a term
 
-$hash{TEXT[,HASH}
+$hash{TEXT[,HASH]}
     unique ID for ``TEXT`` string using the hashing algorithm specified by
-    ``HASH`` which defaults to MD5 hashing algorithm. Currently, this command
-    only supports MD5 which yields a 128-bit hash sequence of 32 hexadecimal
+    ``HASH`` which defaults to MD5. Currently, this command only supports
+    MD5 which yields a 128-bit hash sequence serialised as 32 hexadecimal
     characters.
```

Comment:
If we start talking about security, we probably shouldn't use md5 at all. There's an advantage to having warning that the algorithm is going to change, as well. So perhaps dropping the default entirely would be easiest (although there's something a little weird about having an option with only one valid choice, which nonetheless has to be provided).